pkg/cmd/get/environment.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright © 2021 The LitmusChaos Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package get
+
+import (
+	"fmt"
+	"github.com/litmuschaos/litmusctl/pkg/apis/environment"
+	"github.com/litmuschaos/litmusctl/pkg/utils"
+	"github.com/spf13/cobra"
+	"os"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+	"time"
+)
+
+var ChaosEnvironmentCmd = &cobra.Command{
+	Use:   "chaos-environment",
+	Short: "Get Chaos Environment within the project",
+	Long:  `Display the Chaos Environments within the project with the targeted id `,
+	Run: func(cmd *cobra.Command, args []string) {
+		credentials, err := utils.GetCredentials(cmd)
+		utils.PrintError(err)
+
+		projectID, err := cmd.Flags().GetString("project-id")
+		utils.PrintError(err)
+
+		if projectID == "" {
+			utils.White_B.Print("\nEnter the Project ID: ")
+			fmt.Scanln(&projectID)
+
+			for projectID == "" {
```

Comment:
Can we use `if` here?

Reply:
I picked it from experiments get command so the same code is repeated there but doesn't make any sense to put for and then exit anyway. If would be a better choice